README.md

```diff
@@ -1,107 +1,149 @@
 # Overview
```

Comment:
Maybe we want to move most of the examples to a `Tutorial.md` or something? Our `README.md` is pretty long now. I feel like the content in your different language examples is helpful, but maybe we don't need all of it in the README itself.

As I understand it, `README.md` is intended as a *summary* document, containing:

* What is this?
* How do I install this?
* Show me it doing something.
* Where do I get the documentation?
* Where do I request help?
* How do I contribute?

I'd suggest having one shell based example, with output (maybe `bap -dbir` of some specific function or something, but doesn't much matter) with a link to the tutorial. The rest of the questions seem more or or less covered, except contributing, which I'll mark down where you mention it.